.github/workflows/build.yaml

```diff
@@ -0,0 +1,62 @@
+name: build
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - '**'
+  pull_request: {}
+
+jobs:
+  test-n-build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+        python-version:
+          - 3.6
```

Comment:
We need ^python3.6.1 (for pre-commit)

Reply:
Is not available from actions/setup-python - https://github.com/actions/python-versions/blob/master/versions-manifest.json#L332-L494

Reply:
```
Run actions/setup-python@v2
Successfully setup CPython (3.6.10)
```

Reply:
okay  thanks